docs/learning/serializing-xml-schemas.md

```diff
@@ -49,3 +49,23 @@ We can now use the file system API to write the xml to file.
 ```ts
 [[include:Serialize_Schema_To_XML_Write]]
 ```
+
+## Schemas from an iModel
+
+First use the SchemaLoader from the [@itwin/ecschema-metadata](https://www.itwinjs.org/reference/ecschema-metadata/) package to create a SchemaContextEditor that knows about the schemas in the iModel
```

Comment:
`SchemaContextEditor` is `alpha` - it shouldn't appear in documentation or example code.

Reply:
It was already used in the samples and we are finally getting started working on it again and getting some people to prototype with it so I'd like to leave the references in there with the knowledge that we will probably bring it up to beta once we have a bit more work done and a bit more feedback on it

Reply:
Please promote it ASAP, broken links in docs are bad.